gradio/routes.py

```diff
@@ -192,6 +192,10 @@ def build_resource(path: str):
             return FileResponse(build_file)
         raise HTTPException(status_code=404, detail="Build file not found")
 
+    @app.get("/favicon.ico")
+    async def favicon():
+        return FileResponse(app.blocks.favicon_path)
```

Comment:
I am not sure about this but weren't we doing some checks to file paths to not access to different paths within user's computer?
like `os.some_function`